extensions/theme-seti/build/update-icon-theme.js

```diff
@@ -11,29 +11,31 @@ let https = require('https');
 let url = require('url');
 
 // list of languagesId not shipped with VSCode. The information is used to associate an icon with a language association
+// Please try and keep this list in alphabetical order! Thank you.
 let nonBuiltInLanguages = { // { fileNames, extensions  }
-	"r": { extensions: ['r', 'rhistory', 'rprofile', 'rt'] },
 	"argdown": { extensions: ['ad', 'adown', 'argdown', 'argdn'] },
+	"elixir": { extensions: ['ex'] },
 	"elm": { extensions: ['elm'] },
-	"ocaml": { extensions: ['ml', 'mli'] },
-	"nunjucks": { extensions: ['nunjucks', 'nunjs', 'nunj', 'nj', 'njk', 'tmpl', 'tpl'] },
-	"mustache": { extensions: ['mustache', 'mst', 'mu', 'stache'] },
 	"erb": { extensions: ['erb', 'rhtml', 'html.erb'] },
-	"terraform": { extensions: ['tf', 'tfvars', 'hcl'] },
-	"vue": { extensions: ['vue'] },
-	"sass": { extensions: ['sass'] },
-	"puppet": { extensions: ['puppet'] },
-	"kotlin": { extensions: ['kt'] },
-	"jinja": { extensions: ['jinja'] },
-	"haxe": { extensions: ['hx'] },
-	"haskell": { extensions: ['hs'] },
+	"github-issues": { extensions: ['github-issues'] },
 	"gradle": { extensions: ['gradle'] },
-	"elixir": { extensions: ['ex'] },
+	"godot": { extensions: ['gd', 'godot', 'tres', 'tscn'] },
```

Comment:
`godot` is the only addition to this list; all other changes in this file are just alphabetization